eng/pipelines/templates/CrossFramework_Tests_On_Windows.yml

```diff
@@ -0,0 +1,75 @@
+steps:
+- task: PowerShell@2
+  displayName: "Print Environment Variables"
+  inputs:
+    targetType: "inline"
+    script: |
+      Write-Host "##vso[build.updatebuildnumber]$env:FullVstsBuildNumber"
+      Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
+
+- task: PowerShell@1
+  displayName: "Run Configure.ps1"
+  inputs:
+    scriptName: "$(Build.Repository.LocalPath)\\configure.ps1"
+    arguments: "-Force -CleanCache"
+
+- task: MSBuild@1
+  displayName: "Restore for VS2019"
+  inputs:
+    solution: "build\\build.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/t:RestoreVS /p:BuildNumber=$(BuildNumber) /p:BuildRTM=false /v:m"
+
+- task: MSBuild@1
+  displayName: "Run Cross Verify Tests (continue on error)"
+  continueOnError: "true"
+  inputs:
+    solution: "build\\build.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/t:CrossVerifyTests /p:BuildRTM=false /p:BuildNumber=$(BuildNumber) /p:TestResultOutputFormat=xml"
+    maximumCpuCount: true
+  condition: "and(succeeded(), eq(variables['IsOfficialBuild'], 'true'))"
+
+- task: MSBuild@1
+  displayName: "Run Cross Verify Tests (stop on error)"
+  continueOnError: "false"
+  inputs:
+    solution: "build\\build.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/t:CrossVerifyTests /p:BuildRTM=false /p:BuildNumber=$(BuildNumber) /p:TestResultOutputFormat=xml"
+    maximumCpuCount: true
+  condition: "and(succeeded(), not(eq(variables['IsOfficialBuild'], 'true')))"
+
+- task: DownloadBuildArtifacts@0
+  displayName: "Download Build artifacts"
+  inputs:
+    artifactName: "$(VsixPublishDir)"
```

Comment:
What is use of downloading this artifact?

Reply:
Good question! This is to get the NuGet.exe from .\artifacts\VS15 folder. The NuGet.exe will be used to run verify command in net5.0 code path(signed with net5.0 APIs and verify using NuGet.exe which is net472, so that's cross framework verification) . And we're not able to get the NuGet.exe in the same way as net472 code path does (compile a certain project and get NuGet.exe under bin folder).

Reply:
Great point @zivkan ! The DownloadBuildArtifacts task is removed.

The NuGet.exe is needed, there is a "Run Configure.ps1" in eng\pipelines\templates\CrossFramework_Tests_On_Windows.yml. So it creates artifacts itself and doesn't need to download artifacts. Sorry for the previous incorrect answer to your question @erdembayar !